Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.csproj

```diff
@@ -101,8 +82,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.0.0-rc.2" />
-    <PackageReference Include="FSharp.Core" />
```

Comment:
This was needed for Plotly. Can you add this back?

Reply:
All samples in `Charts with XPlot.ipynb` seem working fine even without this package.